### ISSUE TITLE
redmine 1.3.x support

### ISSUE THREAD

**Issue by ghost** [fernandokosh/redmine_time_tracker#66]
Hi,

Are there any plans to support redmine 1.3.x?

When I try to use it, I get this error:

```
ActionController::RoutingError in Welcome#index

Showing vendor/plugins/redmine_time_tracker/app/views/time_trackers/_embed_menu.rhtml where line #52 raised:

No route matches {:controller=>"time_trackers", :action=>"index"}

Extracted source (around line #52):

49: 
50: <% if User.current.allowed_to?(:log_time, nil, :global => true) or User.current.allowed_to?(:view_others_time_trackers, nil, :global => true) %>
51:     <span>
52:         <%= link_to '',
53:             { :controller => 'time_trackers', :action => 'index' },
54:             { :class => 'icon-action icon-list-action', :title => l(:list_time_trackers) }
55:         %>

Trace of template inclusion: vendor/plugins/redmine_time_tracker/app/views/time_trackers/_update_menu.rhtml, app/views/layouts/base.html.erb
```

Thanx,

Ionutz

**Comment by delaitre** [comment]
According to #64, the plugin is working on redmine 1.3.0. I've not tested myself though.

I've noticed on the redmine forum that some people have problem with the plugin and redmine 1.3. I have no free time to work on it however.

Moreover, we've switch from redmine to chiliproject in my company. So if I found time to rework on the plugin, this will be tested on chiliproject, not redmine (I'll will probably need to rename the plugin...). If anyone wants to take maintership of the plugin, I'll be very happy to support him as the project is not actively develop anymore...

**Comment by ghost** [comment]
The error was on trunk (future 1.4), not 1.3.0.

**Comment by delaitre** [comment]
Ok, I reopen the issue then...

**Comment by kotashiratsuka** [comment]
I needed route change as follows. 

<pre>
Index: config/routes.rb
===================================================================
--- config/routes.rb    (revision 8866)
+++ config/routes.rb    (working copy)
@@ -409,6 +409,7 @@
 
   map.connect 'robots.txt', :controller => 'welcome',
               :action => 'robots', :conditions => {:method => :get}
+  map.connect ':controller/:action/:id'
 
   # Used for OpenID
   map.root :controller => 'account', :action => 'login'
</pre>

**Comment by delaitre** [comment]
I'm not a web/rails guru and I don't really know what a route is. Could you explain what your change does and why it's needed now while it was working before? Is adding a route safe or can it lead to security issue? Is there other way to fix the problem?

Thanks in advance!

**Comment by cforce** [comment]
The route maps urls to controller methods in rails, comparable to servlets in java.

Is this routes.rb (in fact redmine 1.4 support now part of your master)

Seems like, according to https://github.com/delaitre/redmine_time_tracker/pull/72

So this issue is closed and redmine 1.4 is supported?

**Comment by delaitre** [comment]
Yes it works (the development version) with Redmine 1.4. Thank you for pointing me this already fixed bug!
